src/components/v2/index.tsx

```diff
@@ -148,12 +147,19 @@ function RouterComponent({ envType }) {
     return (
         <React.Fragment>
             {isLoading && <DetailsProgressing loadingText="Please wait…" size={24} fullHeight />}
-
-            {!isLoading && errorResponseCode && (
-                <div className="dc__loading-wrapper">
-                    <ErrorScreenManager code={errorResponseCode} />
-                </div>
-            )}
+            {
+                errorResponseCode === 404 ?
```

Comment:
can we change it to simple if else as multiple ternary reduce readability

Reply:
Done